File: LeetCode/Search/JS/Leetcode1408-string-matching-in-an-array.js
var stringMatching = function (words) {
    let [ans, n] = [[], words.length];
    for (let i = 0; i < n; i++) {
        for (let j = 0; j < n; j++) {
            if (i != j && words[j].search(words[i]) != -1) {
                ans.push(words[i]);
                break;
            }
        }
    }
    return ans;
};

File: LeetCode/Search/Golang/Leetcode1408-string-matching-in-an-array.go
package sbgo

import "strings"

func stringMatching(words []string) []string {
	ans, n := []string{}, len(words)
	for i := 0; i < n; i++ {
		for j := 0; j < n; j++ {
			if i != j && strings.Contains(words[j], words[i]) {
				ans = append(ans, words[i])
				break
			}
		}
	}
	return ans
}
